scripts/config.yaml

```diff
@@ -0,0 +1,144 @@
+    runtime="krun"
+    configuration="configuration"
+    pluginid=cri
+    if grep -q "version = 2\>" $containerd_conf_file; then
+      pluginid=\"io.containerd.grpc.v1.cri\"
+    fi
+
+    runtime_table="plugins.${pluginid}.containerd.runtimes.$runtime"
+    runtime_type="io.containerd.$runtime.v2"
+    options_table="$runtime_table.options"
+    config_path=""
+    if grep -q "\[$runtime_table\]" $containerd_conf_file; then
+        echo "Configuration exists for $runtime_table, overwriting"
+        sed -i "/\[$runtime_table\]/,+1s#runtime_type.*#runtime_type = \"${runtime_type}\"#" $containerd_conf_file
+    else
+        cat <<-EOT | tee -a $containerd_conf_file
+          [$runtime_table]
+          runtime_type = "${runtime_type}"
+          privileged_without_host_devices = true
+          pod_annotations = ["app.kontain.*"]
+    EOT
+    sed -e "s/^[ \t]*//" -i $containerd_conf_file
+    fi
+
+    [ -c /root/dev/kvm ] && chmod 666 /root/dev/kvm
+
+    echo "restart containerd"
+    systemctl daemon-reload
+    systemctl restart containerd
+
+    sleep infinity
+kind: ConfigMap
+metadata:
+  labels:
+    app: kontain-init
+  name: kontain-install-entrypoint
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: kontain-init
+  name: kontain-node-initializer
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: kontain-init
+  template:
+    metadata:
+      labels:
+        app: kontain-init
+        name: kontain-node-initializer
+    spec:
+      containers:
+        - command:
+            - /scripts/entrypoint.sh
+          env:
+            - name: ROOT_MOUNT_DIR
+              value: /root
+            - name: CONTAINERD_CFG_DIR
+              value: /etc/containerd
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          image: centos:7
+          name: node-initializer
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: /root
+              name: root-mount
+            - mountPath: /scripts
+              name: entrypoint
+            - mountPath: /etc/crio/
+              name: crio-conf
+            - mountPath: /etc/containerd/
+              name: containerd-conf
+            - mountPath: /var/run/dbus
+              name: dbus
+            - mountPath: /run/systemd
+              name: systemd
+      volumes:
+        - hostPath:
+            path: /
+          name: root-mount
+        - configMap:
+            defaultMode: 484
+            name: kontain-install-entrypoint
+          name: entrypoint
+        - hostPath:
+            path: /etc/crio/
+          name: crio-conf
+        - hostPath:
+            path: /etc/containerd/
+          name: containerd-conf
+        - hostPath:
+            path: /var/run/dbus
+          name: dbus
+        - hostPath:
+            path: /run/systemd
+          name: systemd
```

Comment:
I vaguely remember trying to remove this one and trying to call systemctl to restart containerd, and there were issues since the daemonset runs inside the privileged container.  That's likely why its there.

But for AWS, if everything is AMI-based, then we can likely get rid of a lot of steps specifically for AWS.

We will still need it for GKE.